### ISSUE TITLE
Privilege creation fails

### ISSUE THREAD

**Issue by 5obol** [datadrivers/terraform-provider-nexus#46]
Nexus provider version: v1.6.0

My TF code:
```hcl
resource "nexus_privilege" "nx-repository-view-apt-apt-proxy-bionic-read" {
  name = "nx-repository-view-apt-apt-proxy-bionic-read"
  description = "Read privilege for apt-proxy-bionic repository views"
  type = "repository-view"
  format = "apt"
  repository = "apt-proxy-bionic"
  actions = [
    "read"
  ]
}
```
TF plan:
 ------------------------------------------------------------------------
An execution plan has been generated and is shown below.
```shell
Resource actions are indicated with the following symbols:
  + create
Terraform will perform the following actions:
  # nexus_privilege.nx-repository-view-apt-apt-proxy-bionic-read will be created
  + resource "nexus_privilege" "nx-repository-view-apt-apt-proxy-bionic-read" {
      + actions     = [
          + "read",
        ]
      + description = "Read privilege for apt-proxy-bionic repository views"
      + format      = "apt"
      + id          = (known after apply)
      + name        = "nx-repository-view-apt-apt-proxy-bionic-read"
      + repository  = "apt-proxy-bionic"
      + type        = "repository-view"
    }
Plan: 1 to add, 0 to change, 0 to destroy.
```
------------------------------------------------------------------------
TF apply (fails with the error):
```shell
nexus_privilege.nx-repository-view-apt-apt-proxy-bionic-read: Creating...
Error: could not create privilege "nx-repository-view-apt-apt-proxy-bionic-read": HTTP: 400, [ {
  "id" : "PARAMETER format",
  "message" : "may not be empty"
} ]
  on privileges.tf line 3, in resource "nexus_privilege" "nx-repository-view-apt-apt-proxy-bionic-read":
   3: resource "nexus_privilege" "nx-repository-view-apt-apt-proxy-bionic-read" {
```
Can’t see reason why this would fail. There is no “ID” field or “Message”. Can’t find these in a code of groovy for privilege entity as well. Perhaps a bug that can be addressed?

**Comment by Nosmoht** [comment]
Hi @5obol ,

will check that.

**Comment by Nosmoht** [comment]
I've found the bug. Will release a new release soon.

**Comment by Nosmoht** [comment]
Hi @5obol ,

this should be fixed with [v1.6.1](https://github.com/datadrivers/terraform-provider-nexus/releases/tag/v1.6.1).

**Comment by 5obol** [comment]
Great. Updated to use the latest provider and it works smooth! Thanks for the effort.
